### ISSUE TITLE
Downloads with invalid characters in the workbook name fail

### ISSUE THREAD

**Issue by r-richmond** [tableau/server-client-python#165]
When downloading workbooks with a name like `Option A or B?` it fails with the error `OSError(22, 'Invalid argument')` on windows because question marks aren't allowed in file names. (Note: when downloading through the browser it converts characters like this to safe ones.. i.e. `>` becomes `_`)

Looking at the [code](https://github.com/tableau/server-client-python/blob/8e6c9078c0f5894e034393f7fcbc9f14f10c5838/tableauserverclient/server/endpoint/workbooks_endpoint.py#L91) I don't see an easy way to overwrite the downloaded workbook name.

Two Requests: 
  1) adding an optional string parameter to set the downloaded workbook name.
  2) adding an optional boolean parameter to download the workbook name the way the browser / server handles it.

(One of these options should take precedence over the other as they are mutually exclusive but both options would be useful).

**Comment by t8y8** [comment]
Thanks for the report, I can totally see how the bug would arise.

I think something like option 1 shouldn't be too difficult (But I'd like to just sanitize the names by default anyway)

**Comment by r-richmond** [comment]
sounds like you want to implement 2 by default and add an optional parameter for 1 which sounds great to me.

**Comment by RussTheAerialist** [comment]
2 is already supported.  If the filepath argument is not a directory, we will save it as the file specified in filepath.

```python
server.workbooks.download(wb, filepath="C:\\name.twb")  # will download to c:\name.twb
```

Need to check the docs to see if that's reflected in them.

**Comment by RussTheAerialist** [comment]
#166 fixed this and is included in v0.4.x

**Comment by sakethoppula** [comment]
How to disable auto sanitize. I have to download datasources with '(' and '&' in its name
